src/mongocxx/hint.hpp

```diff
@@ -70,8 +60,7 @@ class MONGOCXX_API hint {
     MONGOCXX_INLINE operator bsoncxx::document::value() const;
```

Comment:
Actually, no, these should stay views.  We only need to examine these values, we're not holding onto them, so it's fine if we are passed temporary values that convert to views.